package.json

```diff
@@ -39,7 +39,8 @@
     "socket.io": "^2.0.1",
     "socketio-file-upload": "^0.6.0",
     "universal-analytics": "^0.4.13",
-    "winston": "^2.3.1"
+    "winston": "^2.3.1",
+    "winston-slack-webhook": "billbitt/winston-slack-webhook"
```

Comment:
should this be billbitt?

Reply:
I made my own fork to fix an issue with the package, and submitted a PR to the main package.  The PR was accepted, so will update this to point to main package in next update.